### DISCUSSION TITLE
Have we test the prefetch to optimize some dense loops?

### DISCUSSION THREAD

**Discussion by wangzhen11aaa** [databendlabs/databend#4749] [Q&A] [answered]
Hi, All

Methods of software prefetching
Compiler directed prefetching

Reference:
https://docs.rs/prefetch/latest/prefetch/
https://en.wikipedia.org/wiki/Cache_prefetching

For discussion. Not so important.

**Comment by Xuanwo** [comment, marked as answer]
@sundy-li Can you take a look at this question? Thanks!

**Reply by sundy-li** [reply]
Sorry for the delay. We did not have this optimization currently.

**Reply by Xuanwo** [reply]
Kindly remind for @wangzhen11aaa

**Reply by wangzhen11aaa** [reply]
@Xuanwo  get it.
